--- lttng-tools-2.7.0/extras/bindings/swig/python/Makefile.am.orig	2015-10-03 22:01:06.492768400 +0200
+++ lttng-tools-2.7.0/extras/bindings/swig/python/Makefile.am	2015-10-03 22:02:08.966099112 +0200
@@ -10,7 +10,7 @@
 MAINTAINERCLEANFILES = lttng_wrap.c lttng.py
 
 nodist__lttng_la_SOURCES = lttng_wrap.c
-_lttng_la_LDFLAGS = -module
+_lttng_la_LDFLAGS = -module -avoid-version
 _lttng_la_LIBADD = 	$(top_builddir)/src/lib/lttng-ctl/liblttng-ctl.la 		\
 			$(top_builddir)/src/common/sessiond-comm/libsessiond-comm.la		\
 			$(top_builddir)/src/common/libcommon.la
